app.js

```diff
@@ -0,0 +1,57 @@
+//Counter for Kate's Guessing Game.
+var numCorrect = 0;
+
+//Question #1 - Asks for user's name.
+var userName = prompt('Hello, What\'s your name?');
+alert('Hello ' + userName + '. Very nice to meet you! Would you mind answering a few questions?');
+console.log('The userName is: ' + userName + ' . Completed first question.');
+numCorrect++;
+
+//Question #2 - Asks the user to guess which country I'm from.
+var countryBorn = prompt('Can you guess what country I was born in?') .toUpperCase();
+  if (countryBorn === 'PHILIPPINES') {
+
+    alert('PHILIPPINES is correct!');
+    console.log(userName + 'has correctly guessed the second question.');
+    numCorrect++;
+  } else if (countryBorn != 'PHILIPPINES') {
+    alert('Good guess but the country I was born in is the Philippines, not ' + countryBorn.charAt(0) + countryBorn.toLowerCase().slice(1) + '.');
+    console.log(userName + 'has incorrectly guessed the second question.');
+ }
+
+//Question #3 - Asks the user to guess what type of shark lives in the Puget Sound.
+var guess = prompt('Another fact about me is that I love sharks. Can you guess which shark species live in the Puget Sound? Choose one: great white shark, mako shark, sixgill shark or the black tip reef shark.');
+  if (guess === 'great white shark') {
+      alert('Although some people have said they\'ve seen a great white in the Puget Sound, it\'s not likely you will ever see one in your lifetime. The correct answer is sixgill shark.');
+        console.log(userName + ' has incorrectly guessed the third question.');
+  } else if (guess === 'mako shark') {
+        alert('One of the fastest of all shark species, it is unfortunate to say that mako sharks are not local to our waters. The correct answer is sixgill shark.');
+        console.log(userName + ' has incorrectly guessed the third question.');
```

Comment:
Overall indentation and formatting needs a little improvement, though not much. It looks like you're over-indenting some of these lines with four spaces, and you only need two. 

Also, you don't need to indent under a `var` declaration like on lines 23 and 24. The If block can come back to the same level as the var.